### ISSUE TITLE
Guild Bank error

### ISSUE THREAD

**Issue by vandiel01** [Caerdon/CaerdonWardrobe#18]
bag is ok, but gbank gives new error as of this patch 9.1.5

```
1x CaerdonWardrobe\features\GuildBank.lua:13: hooksecurefunc(): GuildBankFrame_Update is not a function
[string "=[C]"]: in function `hooksecurefunc'
[string "@CaerdonWardrobe\features\GuildBank.lua"]:13: in function `handler'
[string "@CaerdonWardrobe\CaerdonWardrobe-v2.11.1.lua"]:1194: in function <CaerdonWardrobe\CaerdonWardrobe.lua:1185>
[string "=[C]"]: in function `LoadAddOn'
[string "@FrameXML\UIParent.lua"]:509: in function `UIParentLoadAddOn'
[string "@FrameXML\UIParent.lua"]:564: in function `GuildBankFrame_LoadUI'
[string "@FrameXML\UIParent.lua"]:2001: in function <FrameXML\UIParent.lua:1287>

Locals:

```

**Comment by Ciolouse** [comment]
Same for me, do not work anymore on guild bank

```
2x CaerdonWardrobe\features\GuildBank.lua:13: hooksecurefunc(): GuildBankFrame_Update is not a function
[string "=[C]"]: in function `hooksecurefunc'
[string "@CaerdonWardrobe\features\GuildBank.lua"]:13: in function `handler'
[string "@CaerdonWardrobe\CaerdonWardrobe-v2.12.0.lua"]:1194: in function <CaerdonWardrobe\CaerdonWardrobe.lua:1185>
[string "=[C]"]: in function `LoadAddOn'
[string "@FrameXML\UIParent.lua"]:509: in function `UIParentLoadAddOn'
[string "@FrameXML\UIParent.lua"]:564: in function `GuildBankFrame_LoadUI'
[string "@FrameXML\UIParent.lua"]:2001: in function <FrameXML\UIParent.lua:1287>
[string "=[C]"]: ?

Locals:
```

**Comment by Caerdon** [comment]
This is resolved in v2.13.0
